lib/navigation/imu_displacement.hpp

```diff
@@ -20,12 +21,11 @@ class ImuDisplacement {
    * @param imu_acceleration kalman filtered single value for acceleration
    * @param imu_timestamp time at which easurement was taken
    */
-  void updateImuDisplacement(const core::Float imu_acceleration,
-                             const core::TimePoint imu_timestamp);
+  void update(const core::Float imu_acceleration, const core::TimePoint imu_timestamp);
 
-  core::Float getImuDisplacement();
+  core::Float getAccelerometerDisplacement() const;
```

Comment:
i don't think `accelerometer_trajectory_estimator.getAccelerometerDisplacement()` reads well?

Surely `accelerometer_trajectory_estimator.getDisplacementEstimate()` is better?

Reply:
Or even just `.getDisplacement()` if you don't like the estimate bit